test/unit/pr_checker.test.js

```diff
@@ -300,4 +302,92 @@ describe('PRChecker', () => {
       assert.deepStrictEqual(logger.logs, expectedLogs);
     });
   });
+
+  describe('checkCommitAfterReview', () => {
```

Comment:
Can you add a stub in ~L43 and check that `checkCommitsAfterReview` is actually called by `checkAll`?